packages/react-liveness/package.json

```diff
@@ -50,10 +50,10 @@
     "@aws-amplify/ui": "6.0.11",
     "@aws-amplify/ui-react": "6.1.5",
     "@aws-sdk/client-rekognitionstreaming": "3.398.0",
-    "@aws-sdk/util-format-url": "^3.410.0",
-    "@smithy/eventstream-serde-browser": "^2.0.4",
-    "@smithy/fetch-http-handler": "^2.1.3",
-    "@smithy/protocol-http": "^3.0.3",
+    "@aws-sdk/util-format-url": "^3.523.0",
+    "@smithy/eventstream-serde-browser": "^2.0.5",
+    "@smithy/fetch-http-handler": "^2.1.5",
+    "@smithy/protocol-http": "^3.0.5",
```

Comment:
Are these the most recent versions available?

Reply:
Those are the versions yarn brought in when I ran yarn install so yes?